try_dask.py

```diff
@@ -5,17 +5,17 @@
 @author: Matthew Rocklin
 '''
 
+from __future__ import division
 import numpy as np
 import matplotlib.pyplot as plt
 import pandas as pd
 import dask
 from sklearn.svm import SVC
 from sklearn import cross_validation
-from dask.compose import daskify, value
+from dask.compose import do, value
 from dask.async import get_sync
 
-D = daskify
-
+@do
 def train_test(reg_param, train_idx, test_idx, X, y):
```

Comment:
We often want these functions to also work concretely.  For this kind of use case I'm a bit against decoration.